.github/workflows/dev_on_push_workflow_main.yml

```diff
@@ -174,6 +174,10 @@ jobs:
 
   deploy-to-testing:
     name: "Deploy to testing"
+    # helm can only run one upgrade at a time,
+    concurrency:
+      group: deploy-to-testing
+      cancel-in-progress: false
```

Comment:
>    # helm can only run one upgrade at a time,

Si helm peut faire que un seul upgrade at a time, ceci ne risque pas d'en autoriser N ?

Reply:
Non, si un 1er job `deploy-to-testing` est en cours et qu'un 2e job est appelé, ce 2e est mis en attente 
Si un 3e job est appelé, le 1er continue de tourner, le 2e est annulé, et le 3e est mis en attente.

> When a concurrent job or workflow is queued, if another job or workflow using the same concurrency group in the repository is in progress, the queued job or workflow will be pending. Any previously pending job or workflow in the concurrency group will be canceled. To also cancel any currently running job or workflow in the same concurrency group, specify cancel-in-progress: true.